app/render.py

```diff
@@ -17,6 +17,35 @@
 ]
 
 
+def get_url_for_alert(alert, alerts):
+    """
+    Gets the url slug for an alert (given the global alerts object obtained from `Alerts.load()`.)
+
+    Note: The alert must be public to have a url slug.
+
+    Will return a date string in the style '3-jun-2021', unless there were already alerts that day, in which case it'll
+    append a 1-indexed counter on to the end eg '3-jun-2021-2'. Note that the first alert of the day never has a count.
+    """
+    alerts_on_this_day = sorted([
+        other_alert
+        for other_alert in alerts.public
+        if other_alert.starts_at_date.as_local_datetime.date() == alert.starts_at_date.as_local_datetime.date()
```

Comment:
This comment is slightly incorrect now I realised `as_url` is on the `AlertDate` class. It would need to be `other_alert.starts_at.as_url == alert.starts_at.as_url`

Whether that is particularly advantageous I'm not sure

Reply:
i think i'll stick to comparing the rich date class as that feels more correct. Though I will change it from `as_local_datetime.date()` to `as_local_date` as we've got the property available already